test/js/promise_s27.t.js

```diff
@@ -0,0 +1,34 @@
+/*---
+includes: []
+flags: [async]
+---*/
+
+var inherits = (child, parent) => {
+    child.prototype = Object.create(parent.prototype, {
+        constructor: {
+        value: child,
+        enumerable: false,
+        writable: true,
+        configurable: true
+        }
+    });
+    Object.setPrototypeOf(child, parent);
+};
+
+function BoxedPromise(executor) {
+    var context, args;
+    this.boxed = new Promise(wrappedExecutor);
```

Comment:
this.boxed is not used, so this statement can be replaced by:

new Promise(wrappedExecutor);